### PR TITLE
Added flexibility for dependency version, and Squiz standards

### PR DESCRIPTION
**Pull request by jojobudiman**
Upgraded memory limit & added flexibility for dependency version especially for larger projects.

### CODE REVIEW

File: phpcs.xml [hunk 1]
```diff
@@ -43,7 +43,7 @@
     <rule ref="Generic.Metrics.NestingLevel">
         <properties>
             <property name="nestingLevel" value="5"/>
-            <property name="absoluteNestingLevel" value="15"/>
+            <property name="absoluteNestingLevel" value="25"/>
```
Comment (PatricNox) [inline phpcs.xml:46]: is this laravel standard?

File: phpcs.xml [hunk 2]
```diff
@@ -156,5 +156,5 @@
 
     <arg name="colors"/>
     <arg value="spv"/>
-    <ini name="memory_limit" value="128M"/>
+    <ini name="memory_limit" value="256M"/>
```
Comment (PatricNox) [inline phpcs.xml:159]: this is the memory usage for running the code sniffer, not the php setting for the project

File: phpcs.xml (hunk 1 above)
Comment (jojobudiman) [reply phpcs.xml:46]: Unfortunately nope. The higher the value, the nested functions will generate more warnings. Unless, if I change the same value as the `nestingLevel`, there will be only errors and warnings will not be generated. Which do you prefer? 🤔

File: phpcs.xml (hunk 2 above)
Comment (jojobudiman) [reply phpcs.xml:159]: Right, I'll change it again to 128. 64 should be decent but it depends on the `php.ini` settings 👍

**Comment by jojobudiman** [comment]
Also forgot to mention the commit desc, I also added the report summary `arg`

File: phpcs.xml [hunk 3]
```diff
@@ -154,7 +157,9 @@
     <exclude-pattern>*/Providers/*</exclude-pattern>
     <exclude-pattern>*/resources/lang/*</exclude-pattern>
 
+    <arg name="report" value="summary"/>
```
Comment (PatricNox) [inline phpcs.xml:160]: Most developers wants to see what the cause is from the output, this doesn't do that. 

Instead you can remove `<arg value="spv"/>` as its not needed to know which rule causes it! :)

File: phpcs.xml [hunk 4]
```diff
@@ -101,12 +102,14 @@
     </rule>
     <rule ref="Squiz.PHP.Eval"/>
     <rule ref="Squiz.PHP.GlobalKeyword"/>
+    <rule ref="Squiz.PHP.Heredoc"/>
```
Comment (PatricNox) [inline phpcs.xml:105]: > confirmed @ laravel/vendor/laravel/framework/src/Illuminate/Cache

thanks!

File: phpcs.xml [hunk 5]
```diff
@@ -101,12 +102,14 @@
...
     <rule ref="Squiz.PHP.LowercasePHPFunctions"/>
     <rule ref="Squiz.PHP.NonExecutableCode"/>
     <rule ref="Squiz.Scope.MemberVarScope"/>
     <rule ref="Squiz.Scope.MethodScope"/>
     <rule ref="Squiz.Scope.StaticThisUsage"/>
     <rule ref="Squiz.WhiteSpace.CastSpacing"/>
+    <rule ref="Squiz.WhiteSpace.ControlStructureSpacing"/>
```
Comment (PatricNox) [inline phpcs.xml:112]: > Confirmed @ laravel/framework/src/Illuminate/Auth/Passwords

Thanks!

File: phpcs.xml [hunk 6]
```diff
@@ -93,6 +93,7 @@
         <exclude name="Squiz.Arrays.ArrayDeclaration.NoKeySpecified" />
         <exclude name="Squiz.Arrays.ArrayDeclaration.KeySpecified" />
     </rule>
+    <rule ref="Squiz.PHP.CommentedOutCode"/>
```
Comment (PatricNox) [inline phpcs.xml:96]: > confirmed @ laravel/vendor/laravel

thanks!

File: phpcs.xml (hunk 3 above)
Comment (PatricNox) [reply phpcs.xml:160]: once you do this, it all checks out and is ready for merge, thanks alot! @jojobudiman

Comment (jojobudiman) [reply phpcs.xml:160]: Okay, thank you for the suggestion! Will change again 😊

Comment (PatricNox) [reply phpcs.xml:160]: remember to remove <arg name="report" value="summary"/>

Comment (jojobudiman) [reply phpcs.xml:160]: Right, I forgot 😂 , there we go! 👍

**Comment by PatricNox** [comment]
@jojobudiman  Thank you so much for this incredible work! These are great changes.

Feel free to check over https://github.com/fossbarrow incase you look for more repositories to contribute to! 😄

**Comment by jojobudiman** [comment]
No worries @PatricNox , will do despite with limited knowledge, I'll do my best! 😊

**Pull request merged**